packages/engine/README.md

```diff
@@ -48,29 +49,28 @@ The [HASH glossary](https://hash.ai/glossary?utm_medium=organic&utm_source=githu
 ## Questions & Support
 We're building a community of people who care about enabling better decision-making through modeling and simulation. Our [support forum](https://hash.community/?utm_medium=organic&utm_source=github_readme_engine) and [HASH community Discord server](https://hash.ai/discord?utm_medium=organic&utm_source=github_readme_engine) (requires login) are both great places to meet other modelers and get help.
 
-## The State of Development 
+## The State of Development
 
-As outlined above, this project is the next-generation of our simulation engine, and differs from the one currently powering [hCore](https://hash.ai/platform/core?utm_medium=organic&utm_source=github_readme_engine) and [hCloud](https://hash.ai/platform/cloud?utm_medium=organic&utm_source=github_readme_engine). It's published here as a pre-release technology preview, and as such the feature-set and codebase should be considered unstable until it's released. That means that there are a number of features you  may use on the HASH platform that at present may not be supported by this project, notably:
-* Python runners, and therefore **Python behaviors** are currently **disabled**. This is a high-priority item for us and will be one of the main items of development focused on in the near future. Much of the implementation is finished and in this repository should exploring it be of interest (Although as it is _not_ completely finished, expect to find bugs).
+As outlined above, this project is the next-generation of our simulation engine, and differs from the one currently powering [hCore](https://hash.ai/platform/core?utm_medium=organic&utm_source=github_readme_engine) and [hCloud](https://hash.ai/platform/cloud?utm_medium=organic&utm_source=github_readme_engine). It's published here as a pre-release technology preview, and as such the feature-set and codebase should be considered unstable until it's released. That means that there are a number of features you may use on the HASH platform that at present may not be supported by this project, notably:
+* Python runners, and therefore **Python behaviors** are currently **disabled**. This is a high-priority item for us and will be one of the main items of development focused on in the near future. Much of the implementation is finished and in this repository should explore it be of interest (Although as it is _not_ completely finished, expect to find bugs).
```

Comment:
Sentence still seems broken/reads strange to me. Appreciate that was the case prior to this PR as well.

Reply:
I updated the sentence, it should be more readable now.